tests/integration/Storefront/Theme/ThemeTest.php

```diff
@@ -196,13 +185,17 @@ public function testChildThemeConfigStructuredFields(): void
         $criteria = new Criteria();
         $criteria->addFilter(new EqualsFilter('name', $name));
 
+        /** @var ThemeEntity $childTheme */
```

Comment:
please do not use var annotations. Especially in tests you can simply assert the type. also see https://github.com/shopware/shopware/blob/trunk/coding-guidelines/core/writing-code-for-static-analysis.md